src/main/java/com/microsoft/azure/functions/ExecutionContext.java

```diff
@@ -39,5 +39,7 @@
      * Returns the trace context.
      * @return the trace context
      */
-	TraceContext getTraceContext();
+    default TraceContext getTraceContext(){
+        return null;
```

Comment:
instead of null, can we return empty strings?  @yojagad  thoughts?

Reply:
Changed this to return a default TraceContext with empty strings and empty collection

Reply:
@pragnagopa  If we will return an object of the interface we will need to default all the functions that we have in TraceContext interface.

Reply:
returning null from here will throw null ref if getTraceContext().getTraceparent() is called. I would prefer returning null/empty string which is what host would pass if there is no tracecontext being set.